_devel/ci/run-container.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+docker run \
+    -it \
+    --mount type=bind,src=/mnt/f/repo-matterfi/opentxs,dst=/home/src,readonly \
+    --mount type=bind,src=/mnt/f/repo-matterfi/docker/_devel/ci,dst=/home/script,readonly \
+    --mount type=bind,src=/home/pgawron/opentxs/output,dst=/home/output \
```

Comment:
Just wondering why in many places there is a path to your home directory?

Reply:
These are utility/development files for local machine, thus allowed myself to have those paths.